BlueprintUICommonControls/Tests/Sources/TextFieldTests.swift

```diff
@@ -68,5 +68,34 @@ class TextFieldTests: XCTestCase {
 
     }
 
+    func test_focus() {
+        // make a fake window so that focus works
+        let window = UIWindow(frame: UIScreen.main.bounds)
+        let view = BlueprintView()
+        window.addSubview(view)
+        window.makeKeyAndVisible()
```

Comment:
Do we need to explicitly tear down the window? I forget if showing the window retains it, and eg would then live beyond this test.

Reply:
(Might be worth copying/pasting this over and leveraging it: https://github.com/squareup/market/blob/main/ios/MarketUITesting/Sources/XCTestCase+AppHost.swift#L22)

Reply:
Nope! The window isn't retained. I added a helper for scoping though.